.github/workflows/validate_and_publish_release.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     steps:
       - name: Download packages
         id: download_packages
-        uses: dawidd6/action-download-artifact@09f2f74827fd3a8607589e5ad7f9398816f540fe # v3.1.4
+        uses: dawidd6/action-download-artifact@80620a5d27ce0ae443b965134db88467fc607b43 # v7
```

Comment:
lol so much for "weekly": https://github.com/iree-org/iree/blob/3e34e03dc897d3710534488e9857c92ebf2994b5/.github/dependabot.yml#L7-L8

https://github.com/iree-org/iree/pull/19689 updated from `2.26.0` to `3.1.4` along with other deps then this created a new PR just for `3.1.4` to `7`?

cc @marbre

Reply:
That's at least _strange_... Let's see if this happens more often and if we need to take action.

Generally bumping v3.1.4 to v7 seems fine as there are now only major releases for this action (versioning scheme was changed with v4, see https://github.com/dawidd6/action-download-artifact/releases/tag/v4.